Makefile

```diff
@@ -121,17 +129,39 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 	rm Dockerfile.cross
 
 .PHONY: build-installer
-build-installer: manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
+build-installer: update-prefix manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
 	mkdir -p dist
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default > dist/install.yaml
+	@echo -e "---" >> dist/install.yaml
+	$(KUSTOMIZE) build config/csi-rbac >> dist/install.yaml
 
 ##@ Deployment
 
 ifndef ignore-not-found
   ignore-not-found = false
 endif
 
+PHONY: update-prefix
+update-prefix: backup-prefix ## Update the kustomization.yaml with the new prefix
+	@echo "Updating prefixes in kustomization.yaml files..."
+	sed -i.bak "s/namePrefix: .*/namePrefix: ${KUSTOMIZE_NAME_PREFIX}/" config/default/kustomization.yaml
+	sed -i.bak "s/namespace: .*/namespace: ${NAMESPACE_PREFIX}/" config/default/kustomization.yaml
+	sed -i.bak "s/namePrefix: .*/namePrefix: ${KUSTOMIZE_CSI_NAME_PREFIX}/" config/csi-rbac/kustomization.yaml
+	sed -i.bak "s/namespace: .*/namespace: ${CSI_NAMESPACE_PREFIX}/" config/csi-rbac/kustomization.yaml
+
+.PHONY: revert-prefix
+revert-prefix: ## Revert the kustomization.yaml to original state
+	@echo "Reverting prefixes in kustomization.yaml files..."
+	@if [ -f config/default/kustomization.yaml.bak ]; then mv config/default/kustomization.yaml.bak config/default/kustomization.yaml; fi
+	@if [ -f config/csi-rbac/kustomization.yaml.bak ]; then mv config/csi-rbac/kustomization.yaml.bak config/csi-rbac/kustomization.yaml; fi
+
+.PHONY: backup-prefix
+backup-prefix: ## Backup the kustomization.yaml files
+	@echo "Backing up kustomization.yaml files..."
+	@cp config/default/kustomization.yaml config/default/kustomization.yaml.bak
+	@cp config/csi-rbac/kustomization.yaml config/csi-rbac/kustomization.yaml.bak
+
```

Comment:
This is unnecessary if we follow the kustomize way of applying patches.

Reply:
This is the flexibility to provide kustomization in the makefile nothing else, if users want to run customization manually they can do it.